std_msgs/Header header
string[] joint_names
float32[] angles
float32 rarm_gripper_angle
float32 larm_gripper_angle
